Explorer/Assets/DCL/InWorldCamera/CameraReelGallery/Components/ReelGalleryPoolManager.cs

```diff
@@ -10,12 +9,17 @@ public class ReelGalleryPoolManager
     {
         private readonly IObjectPool<ReelThumbnailController> reelThumbnailPool;
         private readonly IObjectPool<MonthGridController> reelGridPool;
+
         public ReelGalleryPoolManager(
             ReelThumbnailView reelThumbnailPrefab,
             MonthGridView monthViewPrefab,
             GameObject unusedThumbnailPoolObjectParent,
             GameObject unusedGridPoolObjectParent,
             ICameraReelScreenshotsStorage cameraReelScreenshotsStorageService,
+            int gridLayoutFixedColumnCount,
+            int thumbnailHeight,
+            int thumbnailWidth,
+            bool gridShowMonth,
             int thumbnailDefaultCapacity,
             int thumbnailMaxSize,
             int gridDefaultCapacity,
```

Comment:
As with the others, I would make this a struct for easier readability, but its a suggestion of course, nothing that must be changed